packages/tooling/fast-tooling-react/src/form/controls/control.linked-data.tsx

```diff
@@ -238,16 +238,16 @@ class LinkedDataControl extends React.Component<
                         searchTerm: "",
                     });
                 }
-                // Tab performs an autocompete if there is a single schema it can match to
+                // Tab performs an auto-complete if there is a single schema it can match to
             } else if (e.keyCode === keyCodeTab) {
-                e.preventDefault();
-
                 const normalizedValue = e.currentTarget.value.toLowerCase();
                 const matchedSchema = this.lazyMatchValueWithASingleSchema(
                     normalizedValue
                 );
 
                 if (typeof matchedSchema === "string") {
+                    // prevent navigating away by tab when single schema matched
+                    e.preventDefault();
```

Comment:
```suggestion
                    e.preventDefault();

```

Reply:
I rebased and just made the change at the same time. so done and done